src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int FIGURES_COUNT = 5;
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        final int randomSizeRange = random.nextInt(10) + 1;
+
+        switch (random.nextInt(FIGURES_COUNT)) {
```

Comment:
I'm not a big fan of this approach - although the solution is correct, I think it would be more appropriate to pick a random figure type instead of a number/identifier representing that figure.
You can define an enum that stores all figure types and pick a type in the same manner as you do with colors.
Then, your code will be more understandable:
```
case TRIANGE: ...
case SQUARE: ...
...
```